ste/sender-pageBlob.go

```diff
@@ -223,11 +223,9 @@ func (s *pageBlobSenderBase) Prologue(ps common.PrologueState) (destinationModif
 		destinationModified = true
 	}
 
-	if ps.CanInferContentType() {
```

Comment:
CanInferContentType seems useless now, right? Please remove it if appropriate.

Reply:
Done